books/spiders/books_info.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any
+import re
+
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class BooksInfoSpider(scrapy.Spider):
+    name = "books_info"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def __init__(self, **kwargs: Any):
+        super().__init__(**kwargs)
+        options = webdriver.ChromeOptions()
+        options.add_argument("--headless")
+        self.driver = webdriver.Chrome(options=options)
```

Comment:
Using Selenium within a Scrapy spider can significantly slow down the scraping process due to the overhead of launching a browser. Consider whether Selenium is necessary for this task or if Scrapy alone can handle the page content.